### ISSUE TITLE
Adding the "lang" attribut to the html tag of SORMAS

### ISSUE THREAD

**Issue by cmeyerhzi** [hzi-braunschweig/SORMAS-Project#9602]
### Problem Description

sormas has no language defined in the html tag. this causes some browser translation tools to mistakenly translate the page automatically (if the browser is configured accordingly)
### Proposed Change

to avoid this, the introducing of the "lang" attribut should be considered.

**Comment by kwa20** [comment]
It seems like the browser recognizes the german translated page as "en" and therefore suggests to translate it into german. This would at least explain why "Fall" is ranslated into "Herbst". This however never happened before and it is susicious that this happened after the update to 1.72.1

**Comment by cmeyerhzi** [comment]
this problem already existed in earlier versions. we just could never find an explanation for it. what is correct is that parts of the page are recognized as english. other parts are recognized as turkish. the turkish word "sorma" means "frag nicht" in german or "don't ask" in english. of course, this irritates the users when the translation is activated by mistake.

![image](https://user-images.githubusercontent.com/96547924/175492983-af008794-0efd-4ad1-8fed-ee346b3f3f59.png)
